docs/QuickStart-ThinkingInRelay.md

```diff
@@ -7,7 +7,7 @@ permalink: docs/thinking-in-relay.html
 next: videos
 ---
 
-Relay's approach to data-fetching is heavily inspired by our experience with React. In particular, React breaks complex interfaces into reusable **components**, allowing developers to reason about discrete units of an application in isolation, and reducing the coupling between disparate parts of an application. Even more important is that these components are **declarative**: they allow developers to specify *what* the UI should like look for a given state, and not have to worry about *how* to show that UI. Unlike previous approaches that used imperative commands to manipulate native views (e.g. the DOM), React uses a UI description to automatically determine the necessary commands.
+Relay's approach to data-fetching is heavily inspired by our experience with React. In particular, React breaks complex interfaces into reusable **components**, allowing developers to reason about discrete units of an application in isolation, and reducing the coupling between disparate parts of an application. Even more important is that these components are **declarative**: they allow developers to specify *what* the UI should look like for a given state, and not have to worry about *how* to show that UI. Unlike previous approaches that used imperative commands to manipulate native views (e.g. the DOM), React uses a UI description to automatically determine the necessary commands.
```

Comment:
I didn't know about the "rich diff" button, so my clumsy method for inspecting this kind of thing is to `git hub get 428` (via [this script](https://github.com/wincent/wincent/blob/6891f5effb2ef31606921b44f8bafee865fbab60/roles/dotfiles/files/.shells/bin/git-hub)) to get the PR locally, and then you can do whatever you need, like `git show --word-diff=color` (to show word-level diffs) or `git show -b` (to ignore whitespace changes) etc.